### DISCUSSION TITLE
GraphQL Asset Embedded Meta Info

### DISCUSSION THREAD

**Discussion by mjnewman** [pimcore/pimcore#8395] [Q&A / Support] [answered]
Is there a way to retrieve Embedded Meta Info for an asset via GraphQL?

We're looking for a way to retrieve the ImageHeight and ImageWidth properties for images and for thumbnails.

If there is not a way to access this data directly, is there a way to automatically add it to the Custom Metadata or Properties?

**Comment by weisswurstkanone** [comment, marked as answer]
currently there is no support for that. PRs are welcome!

**Comment by bwaidelich** [comment]
FYI: I started working on this with https://github.com/pimcore/data-hub/pull/686
